index.bs

```diff
@@ -547,12 +773,87 @@ The [=remote end steps=] are:
       <dd>The [=remote end=]’s readiness state.</dd>
 
       <dt>"message"</dt>
-      <dd>An implementation-defined string explaining the [=remote end=]’s readiness state.</dd>
+      <dd>An implementation-defined string explaining the [=remote end=]’s readiness
+   state.</dd>
    </dl>
 
-2. Return [=success=] with data |body|.
+2. Return [=success=] with data |body|
 
-Conformance {#conformance}
-==========================
+### Subscribe ### {#command-session-subscribe}
+
+The <dfn export for=commands>subscribe</dfn> command enables certain events either
+globally or for a set of browsing contexts
+
+Issue: This needs to be generalised to work with realms too
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      ```
+      SubscribeCommand = {
+        method: "session.subscribe",
+        params: SubscribeParameters
+      }
+
+      SubscribeParameters = {
+        events: [text],
```

Comment:
Still learning CDDL, but I believe these need a `*` to indicate multiple occurrences, otherwise it is assumed to be exactly one: https://tools.ietf.org/html/rfc8610#section-3.2